xformers/components/multi_head_dispatch.py

```diff
@@ -59,10 +59,10 @@ def __init__(
         attention: Attention,
         bias: bool = True,
         residual_dropout: float = 0.0,
+        use_separate_proj_weight: bool = True,
         dim_key: Optional[int] = None,
         dim_value: Optional[int] = None,
-        in_proj_container: Optional[InProjContainer] = None,
-        use_separate_proj_weight: Optional[bool] = False,
```

Comment:
this was a nasty default I believe, by default we would use the same weights for q/k/v projections, which for the self attention is probably not a good idea